frontend/ui/widget/textwidget.lua

```diff
@@ -44,7 +48,19 @@ function TextWidget:updateSize()
     else
         self._length = math.ceil(tsize.x)
     end
-    self._height = math.ceil(self.face.size * 1.5)
+    -- Used to be:
```

Comment:
Does this matter?

Reply:
It gives context to the following comments. Shoud i move all that in the commit text, and let the code without references to what it was previously?

Reply:
I don't know how much context is really needed beyond the freetype doc on vertical alignment but I'm not one of those comment haters. :-P